src/page/order-summary.page.ts

```diff
@@ -0,0 +1,14 @@
+import { $, ElementFinder } from 'protractor';
+
+export class OrderSummaryPage {
+  private confirmationText: ElementFinder;
+
+  constructor() {
+    this.confirmationText = $('#center_column > div > p > strong');
+  }
+
+  public async confirmOrder(): Promise<void> {
+    await expect(this.confirmationText.getText())
```

Comment:
The idea with POM is just encapsulating business actions, but, verifications or expects should go on the test class. Please make this function to return a string and make the expect on the test class.